### ISSUE TITLE
Bug: Color Contrast in Community Join Box

### ISSUE THREAD

**Issue by charleshalldesign** [GSA/digitalgov.gov#2144]
**Describe the bug**
Similar to Bug #1955, the “Join by email” link has insufficient contrast.

**Steps to Reproduce**
1. Go to https://digital.gov/communities/accessibility/
2. Inspect ‘Join by email’
3. Observe that the text color #fcfcfc over the background color of #e5a000 has a contrast of 2.2:1 — which should be a minimum of 4.5:1 at the rendered size of 16.96px (per SC 1.4.3)
4. Observe that the element background color of #e5a000 over the section background color of #e6e6e2 has a contrast of 1.8:1 — which should be a minimum of 3:1 (per SC 1.4.11)

**Screenshots**
(attached)
<img width="308" alt="Screen Shot 2020-03-04 at 1 04 41 PM" src="https://user-images.githubusercontent.com/3154739/75908798-d42a7600-5e18-11ea-84bb-30f44670288e.png">

**Comment by jeremyzilar** [comment]
Thank you for making an issue about this @hallmedia 🎉

You are correct, the contrast in that area is not great. Could be better.

### Here are the tokens I am using:
- button color `gold-30v`
- button text color `gray-1`
- box background color `gray-warm-10`

And after looking at the **Magic Number** section [USWDS System Color Guidance](https://designsystem.digital.gov/design-tokens/color/overview/#magic-number) page which says,
> **A magic number of 40+** results in WCAG 2.0 AA Large Text contrast `(example: gray-90 and indigo-warm-50v)`.

### Here is what I should be using:
- button text color `white`
- button color `gold-40v`
- box background color — this either needs to be `0` or I need to increase the button color to at least be 40 steps off from the background

**Comment by maya** [comment]
You need a magic number of 50+ at that font size (17px), you need at least 18.66px and bold to use 40+.

**Comment by charleshalldesign** [comment]
Commenting to address the github-actions bot.
